content/en/platform/corda/4.8/enterprise/node/operating/node-administration.md

```diff
@@ -17,22 +17,29 @@ weight: 145
-to time. You can have logging printed to the console as well by passing the `--log-to-console` command line flag.
-The default logging level is `INFO` which can be adjusted by the `--logging-level` command line argument. This configuration
-option will affect all modules. Hibernate (the JPA provider used by Corda) specific log messages of level `WARN` and above
-will be logged to the diagnostic log file, which is stored in the same location as other log files (`logs` subdirectory
-by default). This is because Hibernate may log messages at WARN and ERROR that are handled internally by Corda and do not
-need operator attention. If they do, they will be logged by Corda itself in the main node log file.
+Corda's logging feature uses a [Log4j 2](https://logging.apache.org/log4j/2.x/) component and an [SLF4J ](https://www.slf4j.org/interface)
+interface as its abstraction layer. The latest Corda logging configuration file can be found on [GitHub](https://github.com/corda/corda/blob/release/os/4.10/config/dev/log4j2.xml).
 
-It may be the case that you require to amend the log level of a particular subset of modules (e.g., if you’d like to take a
-closer look at hibernate activity). So, for more bespoke logging configuration, the logger settings can be completely overridden
-with a [Log4j2](https://logging.apache.org/log4j/2.x) configuration file assigned to the `log4j.configurationFile` system property.
+* By default, node log files are stored to the `logs` subdirectory of the working directory and are rotated from time to time.
+* Passing the `--log-to-console` command line flag logs printing to the console.
+* The default logging level is `INFO` which can be adjusted by the `--logging-level` command line argument. This configuration option will affect all modules.
+  * [Hibernate](http://hibernate.org/orm/)
+    (the JPA provider used by Corda) specific log messages of level `WARN` and above will be logged to the diagnostic log file,
+    which is stored in the same location as other log files (`logs` subdirectory by default). This is because Hibernate may log
+    messages at WARN and ERROR that are handled internally by Corda and do not need operator attention. If they do, they will be
+    logged by Corda itself in the main node log file.
+* Aborted flows are marked with `ERROR` or `TERMINATE`.
 
-Additionally, detailed logging around interactions with the database or HSM can be enabled by setting the `logging-level` to `TRACE` and will
-be printed out in a separate file (name will be prefixed with *details*) in the same location as the other log files. This type of logging is better
-structured to allow for log processing by 3rd party tools. MDC is also enabled for it. Currently, the following statement types are supported:
+### Custom logging
 
+Custom logging settings can be set [using Log4j](https://logging.apache.org/log4j/2.x/manual/configuration.html). The command to configure a custom logging file route is:
```

Comment:
```suggestion
You can set custom logging settings using [Log4j](https://logging.apache.org/log4j/2.x/manual/configuration.html). The command to configure a custom logging file route is:
```
Active voice + a bit of subtle link best practice - in general, it's best to only link the relevant noun